src/compas/geometry/_primitives/torus.py

```diff
@@ -0,0 +1,281 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from compas.geometry import Plane
+
+__all__ = ['Torus']
+
+
+class Torus(object):
```

Comment:
to complete, maybe add here `volume` as well

Reply:
added `volume`and `area`as property